reference/forms/types/options/choice_loader.rst.inc

```diff
@@ -73,3 +73,8 @@ better performance::
             ]);
         }
     }
+
+.. tip::
+
+    To learn how to create a custom choice type with specific loaders, you can
+    read :doc:`create a custom choice type </form/create_custom_choice_type>`.
```

Comment:
```suggestion
    read ":doc:`/form/create_custom_choice_type`".
```

(this will use the page title as label, I like it because it's a real title and it keeps in sync with any changes to the article)

Reply:
Nice :), done!